MobileWallet/Screens/RestoreWallet/AppTableViewCell.swift

```diff
@@ -0,0 +1,194 @@
+//  AppTableViewCell.swift
```

Comment:
`SettingsTableViewCell` may be better, let's talk.

Reply:
Initially, I thought the same. But later I decided that maybe it would be a unified cell for the entire application. Exactly nobody knows this yet, but everything can be

Reply:
You can go for something like `SystemMenuTableViewCell`, but there really is no hint of it becoming a general cell, so you can be specific it's ok.